pkg/apis/provisioning/v1alpha5/provisioner_validation.go

```diff
@@ -114,19 +114,36 @@ func (c *Constraints) validateTaints() (errs *apis.FieldError) {
 // When this function is called, the provisioner's requirments do not include the requirements from labels.
 // Provisioner requirements only support well known labels.
 func (c *Constraints) validateRequirements() (errs *apis.FieldError) {
+	if err := ValidateRequirements(c.Requirements); err != nil {
+		errs = errs.Also(apis.ErrInvalidValue(err, "requirements"))
+	}
+	return errs
+}
+
+func ValidateRequirements(requirements []v1.NodeSelectorRequirement) error {
```

Comment:
Why make a new function and not implement this inside `func (c *Constraints) validateRequirements()`